pkg/security/secl/rules/ruleset.go

```diff
@@ -36,6 +37,22 @@ const (
 	OverridePolicy CombinePolicy = "override"
 )
 
+// OverrideField defines a combine field
+type OverrideField = string
+
+const (
+	OverrideAllFields       OverrideField = "all"
+	OverrideExpressionField OverrideField = "expression"
+	OverrideActionFields    OverrideField = "actions"
+	OverrideEveryField      OverrideField = "every"
```

Comment:
Here is my understanding about the difference between `all` and `every`: for `every`, it overrides every field defined in `override_options` while all other fields are untouched. For `all`, it removes the field that is not in `override_options`. Is that correct?

Reply:
`every` is a field of the rule definition which can be used to adapt the rate limiter for that rule. It is not a mode of the override mechanism, just a field like `expression` or `actions`. So `all` means everything will be overridden, otherwise you specify the fields to override.